shop/static/shop/css/dialog.scss

```diff
@@ -42,7 +42,7 @@ button[shop-dialog-proceed] {
 	display: inline-block;
 }
 
-.shop-address-location {
+.shop-address-city {
```

Comment:
it had to be changed, since that field was renamed in ``shop.models.defaults.AddressModelMixin``.